src/MICore/LaunchOptions.cs

```diff
@@ -433,7 +435,6 @@ public static XmlReader OpenXml(string content)
             settings.IgnoreProcessingInstructions = true;
             settings.IgnoreWhitespace = true;
             settings.NameTable = new NameTable();
-            settings.XmlResolver = null;
```

Comment:
Do you know if there is something else we should do to set this to null? There is a Microsoft-internal FXCop rule in the SDL bar that requires it to be set to null?